src/library.js

```diff
@@ -1897,7 +1836,7 @@ LibraryManager.library = {
     }
   },
   // char* dlerror(void);
-  dlerror__deps: ['$DLFCN'],
+  dlerror__deps: ['$LDSO', '$DLFCN'],
```

Comment:
LDSO is not used in dlerror, yes, but I decided to universally add $LDSO dependency to every `dl*` function, so that there is zero uncertainty whether it might break or not even after potential future changes to `dl*` stuff. After all `dl*` functionality is strongly related to the dynamic linker and unconditionally depending on it is imho ok.

However if it would be better from project maintainers point of view, I will remove LDSO from here.